ccmlib/scylla_node.py

```diff
@@ -1399,6 +1400,18 @@ def hostid(self, timeout=60, force_refresh=False):
             self.error(f"Failed to get hostid using {url}: {e}")
         return None
 
+    def get_highest_supported_sstable_version(self, timeout=60):
```

Comment:
this is not the function @bhalevy was asking to override, i.e. `get_node_supported_sstable_versions` is what exist in `node.py`

I think we should do this implementation in dtest, this is used for upgrades, and only called from dtest.
this logic and api calls are not related to any of the ccm responsibilities

Reply:
Maybe added to https://github.com/scylladb/scylla-dtest/blob/next/tools/rest_clients.py ?

Reply:
yeah that one of the place we have usage of scylla APIs.
and we can extend is with default timeout and such, as needed.

even that I think that each call on it's own, would need to decide if retries are needed, or what's the default timeout.
but yes it's a good think to have them consolidated in one place (I just don't think ccm is that place, I it's beyond it's scope, like it doesn't have any function to do cql connection with a driver)

Reply:
> this is not the function @bhalevy was asking to override, i.e. `get_node_supported_sstable_versions` is what exist in `node.py`
> 
> I think we should do this implementation in dtest, this is used for upgrades, and only called from dtest. this logic and api calls are not related to any of the ccm responsibilities

ok. Should I move the existing `get_node_supported_sstable_versions()`  from ccmlib/node.py to dtest's tools/rest.clients.py ? Its used by only by scylla-dtest/rolling_upgrade_test.py

Reply:
I think it would be better

Not all utility functions fits in ccm

Reply:
Opened new PR on dtest: https://github.com/scylladb/scylla-dtest/pull/4809 will close this PR.